autoload/slime/common.vim

```diff
@@ -25,7 +25,12 @@ function! slime#common#write_paste_file(text)
     call mkdir(paste_dir, "p")
   endif
   let lines = split(a:text, "\n", 1)
-  call writefile(lines, slime#config#resolve("paste_file"), 'b')
+  let paste_file = slime#config#resolve("paste_file")
+  let result = writefile(lines, paste_file, 'bS')
+  if result != 0
+    echoerr "Couldn't write to slime paste file."
+  endif
+  return paste_file
```

Comment:
my initial reaction was `resolve` is probably too fast to benchmark (it's not in some inner loop) 🤔 

but looking at the call sites, it wouldn't hurt to return `paste_file` -- so I think I'm OK with that 👍 

I like checking for a `writefile` error, how about avoid the `result` var? (it's only used for the `if`)

```suggestion
  let paste_file = slime#config#resolve("paste_file")
  if writefile(lines, paste_file, 'bS') != 0
    echoerr "Couldn't write to slime paste file."
  endif
  return paste_file
```